server/sandbox_run_linux.go

```diff
@@ -406,7 +407,21 @@ func (s *Server) runPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 		g.AddAnnotation(k, v)
 	}
 
+	// Add default sysctls given in crio.conf
+	for _, defaultSysctl := range s.config.DefaultSysctls {
+		split := strings.Split(defaultSysctl, "=")
```

Comment:
Should this be a SplitN(defaultSysctl, "=",2)
TO make sure the sysctl does not include a "=", I know it is doubtful, but...

Reply:
Fixed.